plugin/output/postgres/postgres.go

```diff
@@ -238,6 +240,17 @@ func (p *Plugin) Start(config pipeline.AnyConfig, params *pipeline.OutputPluginP
 	}
 	p.pool = pool
 
+	stdBackoff := backoff.NewExponentialBackOff()
+	stdBackoff.Multiplier = 1.2
+	stdBackoff.RandomizationFactor = 0.25
+	stdBackoff.InitialInterval = p.config.Retention_
+	stdBackoff.MaxInterval = p.config.Retention_ * 2
+
+	ctxBackoff := backoff.WithContext(stdBackoff, p.ctx)
+	expBackoff := backoff.WithMaxRetries(ctxBackoff, uint64(p.config.Retry))
+
+	p.backoff = expBackoff
```

Comment:
maybe move this into another function?

Reply:
I doesn't think it's a good idea, we can only move: 
```
stdBackoff := backoff.NewExponentialBackOff()
stdBackoff.Multiplier = 1.2
stdBackoff.RandomizationFactor = 0.25
```

Setting `InitialInterval`, `MaxInterval` and wrapping into `WithMaxRetries` can differ: Splunk doesn't have max retries and different logic of intervals. Seems like premature optimization for me.